lms/static/js/commerce/views/receipt_view.js

```diff
@@ -124,17 +119,16 @@ var edx = edx || {};
          * @return {object} JQuery Promise.
          */
         getReceiptData: function (orderId) {
-            var urlFormat = '/shoppingcart/receipt/%s/';
+            var urlFormat = '/shoppingcart/receipt/{orderId}/';
 
             if (this.ecommerceOrderNumber) {
-                urlFormat = '/api/commerce/v1/orders/%s/';
+                urlFormat = '/api/commerce/v1/orders/{orderId}/';
             } else if (this.ecommerceBasketId){
-                urlFormat = '/api/commerce/v0/baskets/%s/order/';
+                urlFormat = '/api/commerce/v0/baskets/{orderId}/order/';
             }
 
-
             return $.ajax({
-                url: _.sprintf(urlFormat, orderId),
+                url: edx.StringUtils.interpolate(urlFormat, {orderId: orderId}),
```

Comment:
Although StringUtils has been exposed globally for non-RequireJS implementation, because this file does use RequireJS can you please pass it in as explained here http://edx-developer-guide.readthedocs.io/en/latest/conventions/safe_templates.html#javascript-context-and-escaping#id32
Eventually we want to move all JS to use RequireJS, and then we will be removing the globally added edx.StringUtils.